community/settings.py

```diff
@@ -37,6 +37,30 @@
     'django_distill',
 ]
 
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'formatters': {
+        'standard': {
+            'format': '%(asctime)s [%(levelname)s] %(name)s: %(message)s'
+        }
+    },
+    'handlers': {
+        'communityHandler': {
+            'class': 'logging.FileHandler',
+            'filename': './_site/community.log',
+            'formatter': 'standard',
+            'level': 'NOTSET'
```

Comment:
Set the level to INFO please.

Reply:
why? we have some log with `ERROR` level. like here: https://github.com/seeeturtle/community/blob/af7306e3e886d38d230a50efafa790f7a26221cc/gci/config.py#L33

Reply:
This specifies the *minimum* level, and all levels above that are included.

Reply:
ah, I see. I didn't know well, sorry.